spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/core/PubSubConfiguration.java

```diff
@@ -132,19 +132,6 @@ public void initialize(String defaultProjectId) {
     this.fullyQualifiedSubscriptionProperties = Collections.unmodifiableMap(fullyQualifiedProps);
   }
 
-  /**
-   * Returns properties for the specified subscription name and project ID.
-   *
-   * @param name short subscription name
-   * @param projectId subscription project name
-   * @return user-provided subscription properties
-   * @deprecated use {@link #getSubscriptionProperties(ProjectSubscriptionName)} instead.
-   */
-  @Deprecated
```

Comment:
I intend to say `all deprecated methods are removed` in Spring Cloud GCP migration guide as breaking changes are expected in this upgrade.

However, I don't intend to list all removed methods and alternatives because we want the autoconfiguration and property files does work for the customers (Spring Boot doesn't list all removed methods in their migration guide) rather than manually create beans. 

How about your thoughts?

Reply:
As you mentioned, this aligns with what the Spring Boot migration guide states:

```
Classes, methods and properties that were deprecated in Spring Boot 2.x have been removed in this release. Please ensure that you aren’t calling deprecated methods before upgrading.
```
Reference: https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#review-deprecations-from-spring-boot-2x 

I think emulating this should be okay. @meltsufin wdyt?

Reply:
Yes, I think it's fine to remove the deprecated methods in a major release like this one.

Reply:
Cool, then I'll change our migration guide aligned with Spring's.